spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/bedrock/llama/BedrockLlamaChatAutoConfiguration.java

```diff
@@ -44,7 +44,8 @@
 @AutoConfiguration
 @ConditionalOnClass(LlamaChatBedrockApi.class)
 @EnableConfigurationProperties({ BedrockLlamaChatProperties.class, BedrockAwsConnectionProperties.class })
-@ConditionalOnProperty(prefix = BedrockLlamaChatProperties.CONFIG_PREFIX, name = "enabled", havingValue = "true")
+@ConditionalOnProperty(prefix = BedrockLlamaChatProperties.CONFIG_PREFIX, name = "enabled", havingValue = "true",
+		matchIfMissing = true)
```

Comment:
Because, currently,  we are using a single boot starter for all Bedrock providers, the Bedrock models should be [disabled by default](https://docs.spring.io/spring-ai/reference/api/bedrock-chat.html#_enable_selected_bedrock_model).